go/tricorder/metric.go

```diff
@@ -960,7 +960,7 @@ func (v *value) evaluate(s *session) reflect.Value {
 	return result
 }
 
-// AsXXX methods return this value as a type XX.
+// AsBool methods return this value as a type XX.
```

Comment:
This comment is for multiple functions. Either the comment should be detached from the function immediately below, or each function should have a comment with the return type specified.